ui/scss/component/_stripe-card.scss

```diff
@@ -423,3 +423,18 @@ pre {
 .currency-to-use-selector {
   margin: 0 auto;
 }
+
+.getting-card-status__div .card__header--between {
+  justify-content: center;
```

Comment:
Why not just left-justify and truncate?  It looks like the max is 48 characters anyway, and the legacy email text can just be truncated to ellipsis.  When it wraps to 3 lines, the text is horizontally cut off (not shown here).

<img width="256" alt="image" src="https://user-images.githubusercontent.com/64950861/172975426-c0e5990c-6152-44b8-8686-521b2f531098.png">

<sub>Hubert Blaine Wolfeschlegelsteinhausenbergerdorff Sr.</sub>
<img width="318" alt="image" src="https://user-images.githubusercontent.com/64950861/172976161-f6ba83b2-77bb-4a3a-941b-ee9f131ef45a.png">

<img width="721" alt="image" src="https://user-images.githubusercontent.com/64950861/172976328-faf17ecd-90ef-40db-8d8d-30c34355a80d.png">